server/routes/shared/showReferralPresenter.ts

```diff
@@ -65,7 +65,10 @@ export default class ShowReferralPresenter {
   get probationPractitionerTeamDetails(): SummaryListItem[] {
     const { activeTeam } = this
     return activeTeam == null
-      ? []
```

Comment:
nit: i reckon you can collapse this into one branch by having the lines be an array of a single undefined value

{ key: 'Phone', lines: [activeTeam?.telephone] }

Reply:
I'm not sure that'd be very helpful for views that have to consume this property, though - they're not expecting to get `undefined` values in this array.